src/global/project/manifest.rs

```diff
@@ -468,6 +468,29 @@ impl FromStr for Mapping {
     }
 }
 
+#[derive(Default)]
+pub enum ExposedType {
+    #[default]
+    All,
+    Subset(Vec<Mapping>),
+}
+
+impl ExposedType {
+    pub fn from_mapping(mapping: Vec<Mapping>) -> Self {
```

Comment:
```suggestion
    pub fn from_mappings(mappings: Vec<Mapping>) -> Self {
```

Also, couldn't `mappings` contain every mapping, and should then return `Self::All`?

Reply:
technically yes, but the subset containing all is the same as all. And it also cheaper for me to do it rather than collecting all binaries from file system.

It will also not matter on the update, as I will verify it again.